docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md

```diff
@@ -187,6 +196,19 @@ process.
 To successfully authenticate wth Entra ID, you should use the `v2.0` API. This means that
 the `CAMUNDA_IDENTITY_ISSUER_BACKEND_URL` value should end with `/v2.0`.
 
+It's also important to follow the [steps described here](https://learn.microsoft.com/en-us/entra/identity-platform/reference-app-manifest#configure-the-app-manifest) to configure the app manifest and set the [accesstokenAcceptedVersion](https://learn.microsoft.com/en-us/entra/identity-platform/reference-app-manifest#accesstokenacceptedversion-attribute) to `2` like so:    
+
+```json
+	"accessTokenAcceptedVersion": 2,
+```
+
+<!-- TODO: we should either add a text input field on Web Modeler Deploy screen to enter the token scope, or automatically configure it via helm charts-->
+To deploy bpmn diagrams from Web Modeler to Zeebe, Make sure to configure the `ZEEBE_TOKEN_SCOPE` environment variable for the Web Modeler Rest API component. For example, run the following command to add this environment variable in kubernetes environments:
```

Comment:
Adding `ZEEBE_TOKEN_SCOPE` is a workaround for the missing "Scope" field. I opened https://github.com/camunda/web-modeler/issues/8027 for fixing this.

Reply:
@wollefitz will this be rolled out soon? I can remove it from this docs PR.

Reply:
@akeller Yes, it is included in the `8.4.2` release of Web Modeler. Let's remove this from the PR as it is a workaround with the side effect that users are only able to deploy to clusters secured by the configured Entra ID instance. This contradicts Web Modeler's principle of being independent of specific clusters.